.github/scripts/tests/tests.sh

```diff
@@ -28,16 +28,21 @@ RESOURCES_DIR_CRD="${GIT_WORKSPACE}/.github/resources"
 OPENDATAHUB_NAMESPACE="opendatahub"
 RESOURCES_DIR_PYPI="${GIT_WORKSPACE}/.github/resources/pypiserver/base"
 ENDPOINT_TYPE="service"
+DSPO_IMAGE_REF="${DSPO_IMAGE_REF:-""}"
 
 get_dspo_image() {
-  if [ "$REGISTRY_ADDRESS" = "" ]; then
-    # this function is called by `IMG=$(get_dspo_image)` that captures the standard output of get_dspo_image
-    set -x
-    echo "REGISTRY_ADDRESS variable not defined."
-    exit 1
+  if [ ! -z "$DSPO_IMAGE_REF" ]; then
+    echo $DSPO_IMAGE_REF
+  else
+    if [ "$REGISTRY_ADDRESS" = "" ]; then
```

Comment:
```suggestion
    if [ -z "$REGISTRY_ADDRESS" ]; then
```

just for consistency